modules/authentication.js

```diff
@@ -23,8 +25,26 @@ const onLogin = async (email, password, dispatch) => {
         communityStatus: response.data.community_status,
       },
     });
+
+    // auth.signIn(email, password)
+    //   .then(response => {
+    //     dispatch({
+    //       type: AUTHENTICATE,
+    //       payload: {
+    //         authenticated: true,
+    //         userEmail: response.data.email,
+    //         userName: response.data.name,
+    //         userId: response.data.id,
+    //         loginMessage: `Welcome ${response.data.name}`,
+    //         logoutMessage: "",
+    //         showLoginForm: false,
+    //         communityId: response.data.community_id,
+    //         communityStatus: response.data.community_status,
+    //       },
+    //     });
+    //   })
```

Comment:
Maybe we should remove this commented out code? :)

Reply:
removed!